src/main/webapp/scripts/mapscript.js

```diff
@@ -0,0 +1,38 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/* Set Up Map */
+function initMap() {
+  /* 'Hard Coded' origin, Google UK Pancras Square*/
+  const origin = {
+    lat: 51.533364, 
```

Comment:
Maybe put this into a const with the place it is pointing to to be direct, e.g
`const googleUK = {lat: 51.533364, lng: -0.125777}`

Reply:
I left it as origin because we're making it specific for Google at first, but in the greater program, the origin could change so I didn't give it a specific name.